.github/workflows/cuda.yml

```
@@ -123,7 +123,7 @@ jobs:
         which nvcc || echo "nvcc not in PATH!"
 
         git clone https://github.com/AMReX-Codes/amrex.git ../amrex
-        cd ../amrex && git checkout --detach 25.03 && cd -
+        cd ../amrex && git checkout --detach a634b09c6f227cc06952043de799942424125dcf && cd -
```

Try https://github.com/AMReX-Codes/amrex/commit/107db66f8723799cc0a25e45759c4d28a1dfeffe to fetch latest commit before the merge of https://github.com/AMReX-Codes/amrex/pull/4357:
```suggestion
        cd ../amrex && git checkout --detach 107db66f8723799cc0a25e45759c4d28a1dfeffe && cd -
```